docs/tutorial/gatsby-image-tutorial/gatsby-image-tutorial.md

```diff
@@ -0,0 +1,361 @@
+
+```jsx:title=src/pages/index.js
+export const query = graphql`
+  query {
+    file(relativePath: { eq: "headers/headshot.jpg" }) {
+      childImageSharp {
+        fixed(width: 125, height: 125) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+```
+
+There are a couple of things to note here.
+
+### Relative image paths and `gatsby-config.js`
+
+You might expect the relative path to be relative to the file the code sits in, in this case that's index.js. However, that doesn't work. The relative path is actually based on the line of code you put in the `gatsby-source-filesystem` config, which points to `src/data`.
+
+### Image fragments
+
+Another thing to note about this query is how it uses the fragment `GatsbyImageSharpFixed` to return a fixed width and height image. You could also use the fragment `GatsbyImageSharpFluid` which produces scalable images that fill their container instead of fitting specific dimensions. In `gatsby-image`, _fluid_ images are meant for images that don’t have a finite size depending on the screen, where as other images are _fixed_.
+
+The query will return a processed image at the highest depth. That is, the deepest part of the return object. The returned result will be automatically passed into the component and attached to the `data` prop. You can then display the image using JSX to automatically output responsive, highly performant HTML.
+
+To display the image, start by importing the component provided by `gatsby-image`.
+
+```jsx
+import Img from "gatsby-image"
+```
+
+Now you can use it. Note that the key for pointing to the image corresponds to the way in which the image was processed. In this example that is `fixed`.
+
+```jsx
+<Img
+  className="headshot"
+  fixed={data.file.childImageSharp.fixed}
+  alt="headshot"
+/>
+```
+
+Here is the query and usage all put together:
+
+```jsx:title=src/pages/index.js
+import React from "react"
+import { graphql } from "gatsby"
+import Img from "gatsby-image"
+import Layout from "../components/layout"
+
+const HomePage = ({ data }) => {
+  return (
+    <Layout>
+      <Img
+        className="headshot"
+        fixed={data.file.childImageSharp.fixed}
+        alt="headshot"
+      />
+    </Layout>
+  )
+}
+
+export const query = graphql`
+  query {
+    file(relativePath: { eq: "headers/headshot.jpg" }) {
+      childImageSharp {
+        fixed(width: 125, height: 125) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+export default HomePage
+```
+
+## Querying for multiple images from YAML data
+
+Another way to source images is through YAML (or Markdown). This example uses the `gatsby-transformer-yaml` plugin to query the YAML files. More information about that plugin can be found in the [Gatsby plugin library](https://www.gatsbyjs.org/packages/gatsby-transformer-yaml/?=gatsby-transformer-yaml).
+
+Here's an example of a query from a list of conferences in a YAML file:
```

Comment:
```suggestion
Here's an example of a query from a list of conferences in a YAML file with an image for each one:
```
In this section, I think it would help to get to the image part sooner so the tutorial doesn't get side-tracked with YAML query information. It would help mitigate the "forking" question you had earlier, @lstar19.